lib/post_action_creator.rb

```diff
@@ -334,7 +335,7 @@ def create_message_creator
       raw: body,
     }
 
-    if %i[notify_moderators spam].include?(@post_action_name)
+    if %i[notify_moderators spam illegal].include?(@post_action_name)
```

Comment:
It's strange that this doesn't have `notify_users` too; maybe there is a reason for it.

Reply:
I dare not question it. 😅